cache/class-vary-cache.php

```diff
@@ -486,7 +492,7 @@ private static function stringify_groups() {
 	 */
 	public static function set_cookie_expiry( int $expiry ) {
 		if ( $expiry < HOUR_IN_SECONDS ) {
-			trigger_error( sprintf( '%s: cookie expiry must be greater than or equal to 1 hour (%d)', __METHOD__, HOUR_IN_SECONDS ), E_USER_WARNING );
+			trigger_error( sprintf( '%s: cookie expiry must be greater than or equal to 1 hour (%d)', __METHOD__, intval( HOUR_IN_SECONDS ) ), E_USER_WARNING );
```

Comment:
Unfortunately, phpcs is not smart enough to figure out that `HOUR_IN_SECONDS` is integer. I am explicitly using `intval()` to avoid the suppression comment.

Reply:
I would rather the suppression comment since it might be confusing — at first glance, one may wonder why `intval()` is being used on `HOUR_IN_SECONDS`.

Reply:
Fixed in 29f2d17d2